docs/guides/embeds.md

```diff
@@ -0,0 +1,128 @@
+# Embeds
+
+Embeds in Discord are just **embedded content** that you can use to **organize the response of the bot**, with **title**, **image**, **footer**, and embed **color**. And as a benefit, it has a higher message content limit on its description.
+
+## Concord & embeds
+
+**There are multiple ways of sending an embed with Concord**, but in this guide, we are using the `static approach` way.
```

Comment:
**There are multiple ways of sending an embed with Concord**, but in this guide, we are using the "static approach" way.